### PR TITLE
feat(hatchery): add service logs for hatchery kubernetes

### PR DESCRIPTION
**Pull request by bnjjj**
Signed-off-by: Benjamin Coenen <benjamin.coenen@corp.ovh.com>

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/hatchery/kubernetes/types.go [hunk 1]
```diff
@@ -14,11 +15,14 @@ import (
 )
 
 const (
-	LABEL_HATCHERY_NAME = "CDS_HATCHERY_NAME"
-	LABEL_WORKER        = "CDS_WORKER"
-	LABEL_WORKER_MODEL  = "CDS_WORKER_MODEL"
+	LABEL_HATCHERY_NAME  = "CDS_HATCHERY_NAME"
```
Comment (golangci) [inline engine/hatchery/kubernetes/types.go]: don't use ALL_CAPS in Go names; use CamelCase

File: engine/hatchery/kubernetes/types.go [hunk 2]
```diff
@@ -14,11 +15,14 @@ import (
...
+	LABEL_WORKER         = "CDS_WORKER"
```
Comment (golangci) [inline engine/hatchery/kubernetes/types.go]: don't use ALL_CAPS in Go names; use CamelCase

File: engine/hatchery/kubernetes/types.go [hunk 3]
```diff
@@ -14,11 +15,14 @@ import (
...
+	LABEL_WORKER_MODEL   = "CDS_WORKER_MODEL"
```
Comment (golangci) [inline engine/hatchery/kubernetes/types.go]: don't use ALL_CAPS in Go names; use CamelCase

File: engine/hatchery/kubernetes/kubernetes.go [hunk 4]
```diff
@@ -36,6 +36,17 @@ func New() *HatcheryKubernetes {
...
 }
 
+// Init register local hatchery with its worker model
+func (h *HatcheryKubernetes) Init() error {
+
+	if err := hatchery.Register(h); err != nil {
+		return fmt.Errorf("Cannot register: %s", err)
+	}
+
+	go h.routines(context.Background())
```
Comment (fsamin) [inline engine/hatchery/kubernetes/kubernetes.go]: sdk.GoRoutine

**Pull request merged**